mtags/src/main/scala-3/scala/meta/internal/pc/PcCollector.scala

```diff
@@ -448,6 +466,28 @@ abstract class PcCollector[T](
             sel,
             pos.withSpan(selectNameSpan(sel)),
           )
+        /* Workaround for missing span in:
+         * class MyIntOut(val value: Int)
+         * object MyIntOut:
+         *   extension (i: MyIntOut) def <<uneven>> = i.value % 2 == 1
+         *
+         * val a = MyIntOut(1).un@@even
+         */
+        case sel: Select
+            if sel.span.isZeroExtent && sel.symbol.is(Flags.ExtensionMethod) =>
+          scala.util
+            .Try(NavigateAST.untypedPath(sel.span)(using compilatonUnitContext))
```

Comment:
This won't work for case mentioned in the comment, because `sel.span` points to `@@MyInt(1).uneven`,
so path is `Ident :: Apply(MyInt, 1) :: Select :: _`.

We can maybe take span from parent and use it? Also, I'm not sure if untyped will be then even needed here

```

Reply:
Which comment?

Reply:
The one above this case

```scala
class MyIntOut(val value: Int)
object MyIntOut:
  extension (i: MyIntOut) def <<uneven>> = i.value % 2 == 1

val a = MyIntOut(1).un@@even
```